maint-lib/flavors-modified-vs-master.py

```diff
@@ -42,10 +42,14 @@ def _filediff_intersects_sources(filediff, sources):
 
 # Get list of files different from VS_BRANCH
 try:
-    _run_cmd(['git', 'fetch', 'origin', 'master', '--quiet'])
+    _ = _run_cmd(['git', 'fetch', 'origin', 'master', '--quiet'])
```

Comment:
Looks like we should replace origin master by splitting VS_BRANCH then but maybe in another PR.

Reply:
I don't follow.

Reply:
Oh wait. I see. We should be fetching the VS_BRANCH. Fetching origin master might not always update what we need.

Reply:
@BlaineEXE `VS_BRANCH` is `origin master`. I guess what @ErwanAliasr1 is suggesting is to simply re-use `VS_BRANCH` and split its content to extract `origin` and `master`, instead of hardcoding `origin master`. The result is the same, it's just more elegant.

Reply:
@BlaineEXE do you want to handle the reuse of VS_BRANCH here or shall I merge this one and we fix later ?

Reply:
I thought I already did this and pushed it, but I guess not. I'll do it here since the work is partly done.